server/sonar-web/src/main/js/apps/code/app.js

```diff
@@ -19,32 +19,26 @@
  */
 import React from 'react';
 import { render } from 'react-dom';
-import { Provider } from 'react-redux';
-import { Router, Route, useRouterHistory } from 'react-router';
-import { createHashHistory } from 'history';
-import { syncReduxAndRouter } from 'redux-simple-router';
+import { Router, Route, Redirect, useRouterHistory } from 'react-router';
+import { createHistory } from 'history';
 
-import Code from './components/Code';
-import configureStore from './store/configureStore';
+import App from './components/App';
 
-import './styles/code.css';
+window.sonarqube.appStarted.then(options => {
+  const el = document.querySelector(options.el);
 
-const store = configureStore();
-const history = useRouterHistory(createHashHistory)({ queryKey: false });
+  const history = useRouterHistory(createHistory)({
+    basename: window.baseUrl + '/code'
+  });
 
-syncReduxAndRouter(history, store);
-
-window.sonarqube.appStarted.then(({ el, component }) => {
-  const CodeWithComponent = () => {
-    return <Code component={component}/>;
+  const AppWithComponent = (props) => {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)